address_test.go

```diff
@@ -760,3 +760,159 @@ func TestInvalidCashAddreTestVectors(t *testing.T) {
 		}
 	}
 }
+
+func TestDecodeCashAddressSlpMainnet(t *testing.T) {
+	addrStr := "simpleledger:qrkjty23a5yl7vcvcnyh4dpnxxzuzs4lzqvesp65yq"
```

Comment:
Should we have a test for all uppercase?

Reply:
ok done.